rubocop.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
   s.name = 'rubocop'
   s.version = RuboCop::Version::STRING
   s.platform = Gem::Platform::RUBY
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '>= 2.2.2'
```

Comment:
I would add to @Drenmi's comment by saying that if you are going to require psych 3.1.0, you should probably require ruby 2.5.0 or greater as that seems to be when [ruby core made the switch](https://github.com/ruby/ruby/commit/6d77e28763ed17f75edf3b4072701b4dbd7644bb).